cmd/sriov-network-config-daemon/start.go

```diff
@@ -162,6 +162,10 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	} else {
 		glog.Fatalf("Failed to fetch node state %s, %v!", startOpts.nodeName, err)
 	}
+	// Return an error if the platform is not Baremetal or OpenStack
+	if platformType != utils.Baremetal && platformType != utils.VirtualOpenStack {
```

Comment:
This should be re-visited, because Baremetal is our defaults and this statement will be always True